prisma/migrations/20241128151318_created_link_statistics_table/migration.sql

```diff
@@ -0,0 +1,12 @@
+-- CreateTable
+CREATE TABLE `LinkStatistic` (
+    `id` VARCHAR(191) NOT NULL,
+    `clicks` INTEGER NOT NULL DEFAULT 0,
+    `referrer` VARCHAR(191) NOT NULL DEFAULT '',
```

Comment:
**suggestion**: Allow `referrer` to be nullable to handle cases where the referrer is not available.